playbooks/gce/openshift-node/config.yml

```diff
@@ -22,11 +23,21 @@
             | oo_select_keys(groups['tag_env-host-type-' + oo_env + '-openshift-master'])
             | oo_collect(attribute='ansible_eth0.ipv4.address') }}"
       when: groups['tag_env-host-type-' + oo_env + '-openshift-master'] is defined
+    - name: Setting oo_master_public_ips fact on localhost
+      set_fact:
+        oo_master_public_ips: "{{ hostvars
+            | oo_select_keys(groups['tag_env-host-type-' + oo_env + '-openshift-master'])
+            | oo_collect(attribute='gce_public_ip') }}"
+      when: groups['tag_env-host-type-' + oo_env + '-openshift-master'] is defined
 
 - name: "Configure instances"
   hosts: oo_hosts_to_config
   connection: ssh
   user: root
+
+  vars:
+    tmp_dir: "/tmp/openshift-{{ 9999 | random }}"
```

Comment:
I'm so sorry I suggested this. Your suggestion was WAY better (mktemp).

Basically in the role have a task do a local action to create a tmpdir with mktemp, then register the output, then use that registered output as the rsync location, then ensure absent on it after.

Something like this in the openshift_node role (untested):
- local_action: command mktemp -d openshift-ansible-XXXXXXX
  register: mktemp

- # do the two rsyncs using mktemp.stdout as the value for the tmp dir

- local_action: "file name={{ mktemp.stdout }} state=absent"